src/lib/components/Pagination/Pagination.js

```diff
@@ -69,6 +70,7 @@ Pagination.propTypes = {
     showPrev: PropTypes.bool,
     showNext: PropTypes.bool,
     size: PropTypes.oneOf(["mini", "tiny", "small", "large", "huge", "massive"]),
+    maxTotalResults: PropTypes.number,
```

Comment:
do we still keep the default here: https://github.com/inveniosoftware/react-searchkit/pull/249/files#diff-5e6e91a6aa8b7da0df03350d588f4d4af38b55cbf4de505be53fb628e98304cfR155 ?

Reply:
I think is safer to have a default, then if the backend wants to modify it can do. WDYT?